### PR TITLE
fix: Codegen: Add specific case to handle k8s's 'metav1.Time'

### PR DESCRIPTION
**Pull request by shimonp21**
Attempting to insert it as a json fails because it formats to "2022-01-01T00:01:02" which is not a valid json...

#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Im working now on https://github.com/cloudquery/plugin-sdk/pull/139 which will remove the need for this as otherwise we will have to bundle it infinite number of libraries in the SDK

File: codegen/golang.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/cloudquery/plugin-sdk/schema"
 	"github.com/iancoleman/strcase"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (yevgenypats) [inline codegen/golang.go:15]: we shouldn't import anything plugin specific in the SDK. working on a solution.

Comment (erezrokah) [reply codegen/golang.go:15]: This is the related PR https://github.com/cloudquery/plugin-sdk/pull/139

**Comment by shimonp21** [comment]
Closing in favour of https://github.com/cloudquery/plugin-sdk/pull/139

**Pull request closed without merging**